decidim-admin/config/locales/ca.yml

```diff
@@ -636,4 +636,4 @@ ca:
   time:
     formats:
       datepicker: "%m/%d/%Y"
-      timepicker: "%d/%m/%Y %H:%M"
\ No newline at end of file
+      timepicker: "%d/%m/%Y %H:%M"
```

Comment:
I didn't get a clear answer from them. They said they use single quotes, but nothing more. Will try to press a little more on the subject.